rfc/0012-invalid-values/README.md

```diff
@@ -0,0 +1,107 @@
+* a text description on how the invalid value emerged (like a log message)
+* a last valid value before turning invalid.
+
+Any kind of transformation via expressions including an `INVALID` value produces an `INVALID` value again. 
+
+### INVALID value handling blocks
+
+#### Block-Type `InvalidTableRowFilter`
+This block-type implements the current default behavior: dropping all rows that have invalid values in them. No configuration parameters are available.
+```
+block MyInvalidRowFilter oftype InvalidTableRowFilter {}
+```
+
+#### Block-Type `InvalidTableCellReplacer`
+This block-type allows replacing all invalid values of a column with a valid value.
+```
+block MyInvalidTemperatureReplacer oftype InvalidTableCellReplacer {
+  column: "Temperature";  // refers to table column
+  value: 21;              // expression producing valid value
+}
+```
+
+### Sink Behavior
+In the sinks, we have to reflect the presence of the `INVALID` value type as well. Each sink should bring a default behavior in this regard and allow further configuration modes. The following is an example on how the SQLiteLoader might work.
+
+```
+block CarsLoader oftype SQLiteLoader {
+  table: "Cars";
+  file: "./cars.sqlite";
+  invalidValueStrategy: "meta-column-invalid"
+}
+```
+
+The newly introduced field `invalidValueStrategy` can have the values `"meta-table-mirrored"`, `"meta-table-linked"`, or `"ignore-row"`.
+
+
+#### Meta Table Mirrored
+An additional table is created with the naming convention `<value table name>_META`. In this table, there are columns that mirror the structure of the original table. For each column in the value table, we introduce three meta columns:
+- `<value column name>_isValid`, flag whether the value in the value column is valid or not, 
+- `<value column name>_details`, text for describing how the invalid value emerged,
+- `<value column name>_lastValidValue`, the last valid value before turning invalid. 
+
+Invalid values in the value table are assigned to `null`. As an implication, all value columns must be `nullable`.
+
+In the future, we might add more meta columns in this manner to the meta table.
+
+#### Meta Table Linked
+An additional table is created with the naming convention `<value table name>_META`. In this table, there are columns that mirror the structure of the original table. For each column in the value table, we introduce three meta columns:
+- `<value column name>_isValid`, flag whether the value in the value column is valid or not, 
+- `<value column name>_details`, text for describing how the invalid value emerged,
+- `<value column name>_lastValidValue`, the last valid value before turning invalid. 
+
+Invalid values in the value table are assigned to `null`. As an implication, all value columns must be `nullable`.
+
+In the future, we might add more meta columns in this manner to the meta table.
```

Comment:
These two sections seem to be identical (except for the headline)

Reply:
Whoops, seems like I overwrote something by copy-paste. Readded it.